src/commands/install/InstallCommand.php

```diff
@@ -70,9 +70,11 @@ private function resolveToRelease(RequestedPhar $requestedPhar): SupportedReleas
         );
     }
 
-    private function getDestination(string $pharName, RequestedPhar $requestedPhar, Directory $destination): ?Filename {
-        if ($requestedPhar->hasLocation()) {
-            return $requestedPhar->getLocation();
+    private function getDestination(string $pharName, RequestedPhar $requestedPhar, Directory $destination): Filename {
+        $location = $requestedPhar->getLocation();
+
+        if ($location !== null) {
+            return $location;
         }
```

Comment:
We should probably rather fix the `getLocation` method to not have a nullable return type but rather throw an exception in case it's `null`. Given we have the `hasLocation` method, this would be cleaner.

Reply:
Done